ibis/pandas/execution/tests/test_window.py

```diff
@@ -130,6 +131,63 @@ def custom_window():
     )
 
 
+@pytest.fixture
```

Comment:
i would just put these in the test itself as they are not used anywhere right?

Reply:
sounds good. thanks.